src/ast/operator.rs

```diff
@@ -88,17 +90,11 @@ pub enum BinaryOperator {
     PGRegexIMatch,
     PGRegexNotMatch,
     PGRegexNotIMatch,
-    PGCustomBinaryOperator(PGCustomOperator),
-}
-
-/// PostgreSQL-specific custom operator.
-///
-/// See [CREATE OPERATOR](https://www.postgresql.org/docs/current/sql-createoperator.html) for more information.
-#[derive(Debug, Clone, PartialEq, Eq, Hash)]
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub struct PGCustomOperator {
-    pub schema: Option<String>,
-    pub name: String,
+    /// PostgreSQL-specific custom operator.
+    ///
+    /// See [CREATE OPERATOR](https://www.postgresql.org/docs/current/sql-createoperator.html)
+    /// for more information.
+    PGCustomBinaryOperator(Vec<String>),
```

Comment:
One downside of this approach that I can think of is that it does not restrict the initial parts of the custom operator to valid tokens

Aka this should probably really be something like

```suggestion
    PGCustomBinaryOperator{ ident: Vec<Ident>, op: String),
```

But I think this may be good enough?